helm/prometheus-agent/values.yaml

```diff
@@ -6,17 +6,17 @@ image:
   pullPolicy: "Always"
 
 remoteWrite: []
+#- name: ""     # Name of the remote write integration
```

Comment:
@mcbenjemaa I decided to remove the other authentication use case for now (oauth and so on) because we currently only use basic auth anyway and we cannot use the prometheus operator RemoteWrite struct here because it uses a PasswordSelector that fetches a secret value from a secret and that would mean we would need an extra secret here. I thinki it's easier if we maintain our own struct. We could maybe use the one from prometheus and add only the json annotations but let's start with that and improve if needed

Reply:
I see